lib/components/viewers/pattern-row.tsx

```diff
@@ -73,31 +73,37 @@ class PatternRow extends Component<Props, State> {
 
     const routeName = route.shortName ? route.shortName : route.longName
     const routeColor = getRouteColorBasedOnSettings(route.operator, route)
+    console.log(stopTimes)
+    let style
+    if (routeName && routeName?.length >= 4 && routeName?.length <= 6) {
```

Comment:
I don't think you need the first routeName check! Since you have the question mark